packages/react/src/SubNav/SubNav.tsx

```diff
@@ -6,118 +6,166 @@ import {get} from '../constants'
-      {actions && <div className="SubNav-actions">{actions}</div>}
-    </SubNavBase>
+    <StyledSubNavLinks ref={forwardRef} className={clsx(className, enabled && styles.Links)} {...rest}>
+      {children}
+    </StyledSubNavLinks>
   )
-}
-
-export type SubNavLinksProps = SxProp
+})
+SubNavLinks.displayName = 'SubNav.Links'
 
-const SubNavLinks = styled.div<SubNavLinksProps>`
-  display: flex;
-  ${sx};
-`
+// SubNav.Link
+
+const StyledSubNavLink = toggleStyledComponent(
+  CSS_MODULES_FEATURE_FLAG,
+  'a',
+  styled.a.attrs<StyledSubNavLinkProps>(props => ({
+    className: clsx('SubNav-item', props.selected && 'selected', props.className),
+  }))<StyledSubNavLinkProps>`
+    padding-left: ${get('space.3')};
+    padding-right: ${get('space.3')};
+    font-weight: ${get('fontWeights.semibold')};
+    font-size: ${get('fontSizes.1')};
+    line-height: 20px; //custom value for SubNav
+    min-height: 34px; //custom value for SubNav
+    color: ${get('colors.fg.default')};
+    text-align: center;
+    text-decoration: none;
+    border-top: 1px solid ${get('colors.border.default')};
+    border-bottom: 1px solid ${get('colors.border.default')};
+    border-right: 1px solid ${get('colors.border.default')};
+    display: flex;
+    align-items: center;
 
-type StyledSubNavLinkProps = {
-  to?: To
-  selected?: boolean
-} & SxProp
+    &:first-of-type {
+      border-top-left-radius: ${get('radii.2')};
+      border-bottom-left-radius: ${get('radii.2')};
+      border-left: 1px solid ${get('colors.border.default')};
+    }
 
-const SubNavLink = styled.a.attrs<StyledSubNavLinkProps>(props => ({
-  className: clsx(ITEM_CLASS, props.selected && SELECTED_CLASS, props.className),
-}))<StyledSubNavLinkProps>`
-  padding-left: ${get('space.3')};
-  padding-right: ${get('space.3')};
-  font-weight: ${get('fontWeights.semibold')};
-  font-size: ${get('fontSizes.1')};
-  line-height: 20px; //custom value for SubNav
-  min-height: 34px; //custom value for SubNav
-  color: ${get('colors.fg.default')};
-  text-align: center;
-  text-decoration: none;
-  border-top: 1px solid ${get('colors.border.default')};
-  border-bottom: 1px solid ${get('colors.border.default')};
-  border-right: 1px solid ${get('colors.border.default')};
-  display: flex;
-  align-items: center;
-
-  &:first-of-type {
-    border-top-left-radius: ${get('radii.2')};
-    border-bottom-left-radius: ${get('radii.2')};
-    border-left: 1px solid ${get('colors.border.default')};
-  }
-
-  &:last-of-type {
-    border-top-right-radius: ${get('radii.2')};
-    border-bottom-right-radius: ${get('radii.2')};
-  }
-
-  &:hover,
-  &:focus {
-    text-decoration: none;
-    background-color: ${get('colors.canvas.subtle')};
-    transition: background-color 0.2s ease;
+    &:last-of-type {
+      border-top-right-radius: ${get('radii.2')};
+      border-bottom-right-radius: ${get('radii.2')};
+    }
 
-    .SubNav-octicon {
-      color: ${get('colors.fg.muted')};
```

Comment:
Removed the `SubNav-octicon` stylings since it does not appear that we add this class anywhere, let me know if that is correct.